src/thermo/src/4C_thermo_timint.hpp

```diff
@@ -195,6 +197,9 @@ namespace Thermo
     //! print summary after step
     void print_step() override = 0;
 
+    //! assemble data for runtime writer output
```

Comment:
What do you mean by "assemble" here? I would document what you write and not just repeat the function name.

Reply:
Changed the documentation.